infra/base-images/base-builder/compile_fuzztests.sh

```diff
@@ -50,7 +60,7 @@ FUZZ_TEST_BINARIES=$(bazel query "kind(\"cc_test\", rdeps(${TARGET_FOLDER}, @com
 FUZZ_TEST_BINARIES_OUT_PATHS=$(bazel cquery "kind(\"cc_test\", rdeps(${TARGET_FOLDER}, @com_google_fuzztest//fuzztest:fuzztest_gtest_main))" --output=files)
 
 # Build the project and fuzz binaries
-bazel build $BUILD_ARGS -- ${FUZZ_TEST_BINARIES[*]}
+bazel build $BUILD_ARGS -- ${FUZZ_TEST_BINARIES[*]} ${EXTRA_TARGETS_TO_BUILD}
```

Comment:
Would it be a good idea to reuse the `${:-}` trick here so that we don't have to define `EXTRA_TARGETS_TO_BUILD` as an empty string above in line 46?

Reply:
Good idea, solved! Also fixed a bug where an empty string shouldn't be used in the event the variable is not set, rather we should just use nothing (i.e. no string).